.github/workflows/staging-build-tar.yml

```diff
@@ -33,9 +34,14 @@ jobs:
     - name: Configure AWS Credentials
       uses: aws-actions/configure-aws-credentials@v1
       with:
-        aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
-        aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        aws-access-key-id: ${{ secrets.AWS_STAGING_ACCESS_KEY_ID }}
+        aws-secret-access-key: ${{ secrets.AWS_STAGING_SECRET_ACCESS_KEY }}
         aws-region: us-east-1
+    - name: Required Packages
+      run: |
+        apt-get update
+        apt-get install sudo -y
```

Comment:
Why not just add to the required packages script.

Reply:
The sudo install was necessary in this scenario due to the container being used for ES and Kibana build. The required packages script is tend to be used for all jobs, where this might not be necessary.

Reply:
Ok thanks.